prototype2/Utilities.cmake

```diff
@@ -0,0 +1,69 @@
+#=============================================================================
+# Generate a loadable detector module
+#=============================================================================
+function(create_module module_name)
+  add_library(${module_name} MODULE
+    ${${module_name}_SRC}
+    ${${module_name}_INC})
+  set_target_properties(${module_name} PROPERTIES PREFIX "")
+  set_target_properties(${module_name} PROPERTIES SUFFIX ".so")
+  target_link_libraries(${module_name}
+    ${${module_name}_LIB}
+    ${EFU_COMMON_LIBS}
+    eventlib)
+  enable_coverage(${module_name})
+  install(TARGETS ${module_name} DESTINATION bin)
```

Comment:
This line is ignored (at least under MacOSX) for some reason. I have made a bunch of different attempts at fixing this we no success so far.

Reply:
Which line specifically?
It has worked for me on my mac
We might also consider something like [this](https://github.com/ess-dmsc/h5cpp/blob/master/cmake/OutputDirConfig.cmake)
to standardize these paths.